packages/eslint-plugin-wpcalypso/lib/rules/test/i18n-unlocalized-url.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @file Disallow using unlocalized URL strings
+ * @author Automattic
+ * @copyright 2023 Automattic. All rights reserved.
+ * See LICENSE.md file in root directory for full license.
+ */
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const RuleTester = require( 'eslint' ).RuleTester;
+const rule = require( '../i18n-unlocalized-url' );
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const localizableUrls = [
+	'https://apps.wordpress.com/',
+	'https://automattic.com/cookies/',
+	'https://automattic.com/privacy/',
+	'https://wordpress.com/support/test-article/',
+	'https://wordpress.com/forums/test-thread/',
+	'https://wordpress.com/tos/',
+	'https://wordpress.com/tos/',
```

Comment:
Did we mean to duplicate this?

Reply:
Nope, that was an oversight. Removed it and updated the tests.